test/framework/config_helper.cpp

```diff
@@ -32,6 +32,7 @@ namespace integration_framework {
       std::stringstream ss;
       ss << "host=" << pg_host << " port=" << pg_port << " user=" << pg_user
          << " password=" << pg_pass;
+      logger::log("ITF")->info("Postgres credentials: {}", ss.str());
```

Comment:
mb use logger format instead of string stream?
`logger::log("ITF")->info("Postgres credentials: host={}, port={}", pg_host, pg_port);` and so on